.github/workflows/publish.yml

```diff
@@ -73,7 +73,10 @@ jobs:
           git clone git@github.com:apptainer/apptainer.org.git ~/apptainer.org
           rm -rf ~/apptainer.org/static/docs/user/${GITHUB_REF##*/}
           cp -r _build/html ~/apptainer.org/static/docs/user/${GITHUB_REF##*/}
+          cd ~/apptainer.org/static/docs/user/
+          version=`ls -1 | grep -E "^[0-9\.]+$" | sort -V | tail -n 1`
+          ln -sfn ~/apptainer.org/static/docs/user/${version} ~/apptainer.org/static/docs/user/latest
```

Comment:
A symlink target can be relative to its source.  So change the first part to just `${version}`.

Reply:
It also just occurred to me that we don't really need this code in the 1.0 branch.  This part would be better off as a separate PR to the main branch.